tasks/upgrade.sh

```diff
@@ -19,8 +19,11 @@ chown $tomcat_user:$tomcat_group /tmp/irida.dbbackup
 sudo -u $tomcat_user mv /tmp/irida.dbbackup $db_backup_location/irida-$(date  +"%y-%m-%d-%T").dbbackup
 
 echo "Replacing WAR file..."
-sudo -u $tomcat_user rm $tomcat_location/webapps/irida.war
-sudo -u $tomcat_user curl -o $tomcat_location/webapps/irida.war $war_url
+#removing both expanded folder and war file itself
+sudo -u $tomcat_user rm -rf $tomcat_location/webapps/$irida_url_path
```

Comment:
Could there be a case where `$tomcat_location` and/or `$irida_url_path` are both not set, resulting in this line performing:
```
sudo -u $tomcat_user rm -rf /webapps/
```

Reply:
Added check at the top to ensure that both are set.